src/agent/testrunpublisher.ts

```diff
@@ -4,12 +4,13 @@ import ctxm = require('./context');
 import cm = require('./common');
 import utilities = require('./utilities');
 
-
 var async = require('async');
 var fs = require('fs');
 var path = require("path");
 var xmlreader = require('xmlreader');
 var Q = require('q');
+var archiver = require('archiver');
```

Comment:
IDE doesn't support any refactoring of imports. None of the files have sorted imports.